### DISCUSSION TITLE
Fluid Motion Frames

### DISCUSSION THREAD

**Discussion by PenguinDOOM** [Blinue/Magpie#708] [Ideas]
AMD announced a frame interpolation called Fluid Motion Frames.
Since it is implemented in the driver, it seems that frames can probably be generated without using motion vectors.
AMD plans to release sources for Fluid Motion Frames

**Comment by Blinue** [comment]
I have no intention of implementing frame interpolation. Without motion vector, I am skeptical about how well frame interpolation would work.

**Comment by PenguinDOOM** [comment]
I won't know until it's implemented in the driver, but it depends on how the game screen moves, but if it works well, it's worth adding.
